api/python/notebooks/analysis_demo/comp_bio_embedding_exploration.ipynb

```diff
@@ -35,8 +35,8 @@
     "\n",
     "**Disclaimers** \n",
     "\n",
-    "1. These embeddings were explored in-depth in a [cellxgene](https://github.com/chanzuckerberg/cellxgene) instance and not all of the insights gleaned there will be expanded on here.\n",
-    "2. Most of the following examples utilize UMAP to visualize embeddings in a 2D scatter plot, however as shown [here](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011288) and [here](https://www.cell.com/cell-systems/pdf/S2405-4712(23)00209-0.pdf), biological interpretations from these visualizations may be innacurate.\n",
+    "1. These embeddings were explored in-depth in a [cellxgene](https://github.com/chanzuckerberg/cellxgene) instance and not all the insights gleaned there will be expanded on here.\n",
+    "2. Most of the following examples utilize UMAP to visualize embeddings in a 2D scatter plot, however as shown [here](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011288) and [here](https://www.cell.com/cell-systems/abstract/S2405-4712%2823%2900209-0), biological interpretations from these visualizations may be inaccurate.\n",
```

Comment:
The URL change is due to some issues with how GitHub renders the link. The encoding should fix the issue.